blocks/blocks.go

```diff
@@ -148,20 +148,11 @@ func saveMetrics(blk types.Block) {
 			hist:  metrics.ForDiffLength,
 			value: float64(len(blk.ForDiff)),
 		},
-		{
```

Comment:
why to remove? this is only 2 additional labels and they don't change based on the network state

Reply:
I just wanted to remove metrics that are not important at the moment. But we can keep these two. Reverted